index.js

```diff
@@ -166,6 +167,7 @@ class Logger {
   }
 
   serialize(tags, message, options) {
+    options = aug(this.config, options);
```

Comment:
shouldn't options already be merged at this point? - might need to move blacklist to reporterDefaults